docs/components/concepts/user-task-listeners.md

```diff
@@ -0,0 +1,179 @@
+---
+id: user-task-listeners
+title: "User task listeners"
+description: "User task listeners allow users to react to specific user task lifecycle events by executing custom logic."
```

Comment:
Speaking from a "beginner" perspective -- when you say custom logic do you mean logic rules for standard and custom objects by using decision tables or functions?